samples/plugins/postgres_odbc/connectionBuilder.js

```diff
@@ -12,6 +12,16 @@
     params["UseDeclareFetch"] = "1";
     params["Fetch"] = "2048";
 
+    var odbcConnectStringExtrasMap = {};
```

Comment:
Can you define the string as a const?

`const attributeODBCConnectionStringExtras = "odbc-connect-string-extras";`

Reply:
Fixed